### ISSUE TITLE
ParseError: syntax error, unexpected token "*", expecting end of file

### ISSUE THREAD

**Issue by ddvarpdd** [Universal-Omega/PortableInfobox#39]
MediaWiki 1.36.2, PHP 8.0.12, SQLite
To reproduce, create an Infoboxtest template with following:
```
<infobox>
  <title source="title1">
    <default>test</default>
  </title>
</infobox>
<noinclude>
{{Infoboxtest
|title1 = test
}}
</noinclude>
```
Then just reload the page.

Stack trace:
```
[3ab1819c098e840455977561] /index.php?title=Template:Infoboxtest&action=submit ParseError: syntax error, unexpected token "*", expecting end of file

Backtrace:

from /tmp/lci_5qIZrH(1)
#0 /var/www/wikitest/extensions/PortableInfobox/includes/services/Helpers/PortableInfoboxTemplateEngine.php(89): LightnCandy\LightnCandy::prepare()
#1 /var/www/wikitest/extensions/PortableInfobox/includes/services/Helpers/PortableInfoboxTemplateEngine.php(55): PortableInfobox\Helpers\PortableInfoboxTemplateEngine->getRenderer()
#2 /var/www/wikitest/extensions/PortableInfobox/includes/services/PortableInfoboxRenderService.php(60): PortableInfobox\Helpers\PortableInfoboxTemplateEngine->render()
#3 /var/www/wikitest/extensions/PortableInfobox/includes/services/PortableInfoboxRenderService.php(168): PortableInfoboxRenderService->render()
#4 /var/www/wikitest/extensions/PortableInfobox/includes/services/PortableInfoboxRenderService.php(83): PortableInfoboxRenderService->renderTitle()
#5 /var/www/wikitest/extensions/PortableInfobox/includes/services/PortableInfoboxRenderService.php(182): PortableInfoboxRenderService->renderItem()
#6 /var/www/wikitest/extensions/PortableInfobox/includes/services/PortableInfoboxRenderService.php(35): PortableInfoboxRenderService->renderChildren()
#7 /var/www/wikitest/extensions/PortableInfobox/includes/controllers/PortableInfoboxParserTagController.php(75): PortableInfoboxRenderService->renderInfobox()
#8 /var/www/wikitest/extensions/PortableInfobox/includes/controllers/PortableInfoboxParserTagController.php(129): PortableInfoboxParserTagController->render()
#9 /var/www/wikitest/includes/parser/Parser.php(3971): PortableInfoboxParserTagController->renderInfobox()
#10 /var/www/wikitest/includes/parser/PPFrame_Hash.php(351): Parser->extensionSubstitution()
#11 /var/www/wikitest/includes/parser/Parser.php(2879): PPFrame_Hash->expand()
#12 /var/www/wikitest/includes/parser/Parser.php(1549): Parser->replaceVariables()
#13 /var/www/wikitest/includes/parser/Parser.php(639): Parser->internalParse()
#14 /var/www/wikitest/includes/content/WikitextContent.php(375): Parser->parse()
#15 /var/www/wikitest/includes/content/AbstractContent.php(591): WikitextContent->fillParserOutput()
#16 /var/www/wikitest/includes/EditPage.php(4238): AbstractContent->getParserOutput()
#17 /var/www/wikitest/includes/EditPage.php(4143): EditPage->doPreviewParse()
#18 /var/www/wikitest/includes/EditPage.php(2920): EditPage->getPreviewText()
#19 /var/www/wikitest/includes/EditPage.php(726): EditPage->showEditForm()
#20 /var/www/wikitest/includes/actions/EditAction.php(71): EditPage->edit()
#21 /var/www/wikitest/includes/actions/SubmitAction.php(38): EditAction->show()
#22 /var/www/wikitest/includes/MediaWiki.php(531): SubmitAction->show()
#23 /var/www/wikitest/includes/MediaWiki.php(315): MediaWiki->performAction()
#24 /var/www/wikitest/includes/MediaWiki.php(913): MediaWiki->performRequest()
#25 /var/www/wikitest/includes/MediaWiki.php(546): MediaWiki->main()
#26 /var/www/wikitest/index.php(53): MediaWiki->run()
#27 /var/www/wikitest/index.php(46): wfIndexMain()
#28 {main}
```

**Comment by Universal-Omega** [comment]
Note I am aware of this. This does not currently support PHP8, as MediaWiki itself it not meant to be fully compatible with PHP8 yet.

**Comment by Universal-Omega** [comment]
The issue seems to have been with MediaWiki core on PHP 8, which seems to now be fixed (on latest 1.37 anyways)
